layers/stateless/sl_instance_device.cpp

```diff
@@ -738,11 +738,43 @@ bool StatelessValidation::manual_PreCallValidateGetPhysicalDeviceImageFormatProp
                                  "(%s) but no VkPhysicalDeviceImageDrmFormatModifierInfoEXT in pNext chain.",
                                  string_VkImageTiling(pImageFormatInfo->tiling));
             }
-            if (image_drm_format->sharingMode == VK_SHARING_MODE_CONCURRENT && image_drm_format->queueFamilyIndexCount <= 1) {
-                skip |= LogError("VUID-VkPhysicalDeviceImageDrmFormatModifierInfoEXT-sharingMode-02315", physicalDevice,
+            if (image_drm_format->sharingMode == VK_SHARING_MODE_CONCURRENT) {
+                if (image_drm_format->queueFamilyIndexCount <= 1) {
+                    skip |=
+                        LogError("VUID-VkPhysicalDeviceImageDrmFormatModifierInfoEXT-sharingMode-02315", physicalDevice,
                                  format_info_loc.pNext(Struct::VkPhysicalDeviceImageDrmFormatModifierInfoEXT, Field::sharingMode),
                                  "is VK_SHARING_MODE_CONCURRENT, but queueFamilyIndexCount is %" PRIu32 ".",
                                  image_drm_format->queueFamilyIndexCount);
+                } else if (!image_drm_format->pQueueFamilyIndices) {
+                    skip |= LogError(
+                        "VUID-VkPhysicalDeviceImageDrmFormatModifierInfoEXT-sharingMode-02314", physicalDevice,
+                        format_info_loc.pNext(Struct::VkPhysicalDeviceImageDrmFormatModifierInfoEXT, Field::sharingMode),
+                        "is VK_SHARING_MODE_CONCURRENT, queueFamilyIndexCount is %" PRIu32 ", but pQueueFamilyIndices is null.",
```

Comment:
```suggestion
                        "is VK_SHARING_MODE_CONCURRENT, queueFamilyIndexCount is %" PRIu32 ", but pQueueFamilyIndices is NULL.",
```
For (I think) consistency with the rest of the code base